corehq/messaging/scheduling/models/content.py

```diff
@@ -70,7 +71,7 @@ def send(self, recipient, logged_event, phone_entry=None):
             return
 
         message = self.get_translation_from_message_dict(
-            logged_event.domain,
+            Domain.get_by_name(logged_event.domain),
```

Comment:
instead of fetching the whole domain just to have access to one field how about creating a separate cache for that field?
Could do something like https://github.com/dimagi/commcare-hq/blob/6a8796f1caecc71b342bcd2fce66e7fd246b0754/corehq/apps/domain/models.py#L505-L509

Reply:
Does this and such caching increase the hits on redis too much?
`get_by_name` is also cached though so having just the specific property sounds useful and reduce network load.

Reply:
I tried out this code and then backed out those changes, I think it added more code/complication than it's worth given that domain is already cached.